src/ws/cockpitauth.c

```diff
@@ -761,7 +850,8 @@ cockpit_auth_remote_login_async (CockpitAuth *self,
                                     cockpit_auth_remote_login_async);
 
```

Comment:
Lets not do it now, but keep it in mind. Will comment back up on the documentation.